projects/extension/src/background/index.ts

```diff
@@ -73,6 +73,12 @@ export interface Background extends Window {
     onChainsChanged: (listener: () => void) => () => void
     onSmoldotCrashErrorChanged: (listener: () => void) => () => void
     disconnectTab: (tabId: number) => void
+    getDefaultBootnodes: (chain: string) => string[]
+    saveBootnodes: (chain_id: string, bootnodes: string[]) => void
+    validateAndAddBootnode: (
```

Comment:
Why is there a separate `saveBootnodes` section? Can `validateAndAddBootnode` not also save the bootnode?
Having two functions implies that you add the bootnode to the currently running light client, but this bootnode might not be saved for next time.

Reply:
`saveBootnodes` is actually saving all the given `string[]` into the localStorage while `validateAndAddBootnode` is only checking the validity and not adding in the localstorage. 

> Having two functions implies that you add the bootnode to the currently running light client, but this bootnode might not be saved for next time.

That is correct. User may want to check the validity of the bootnode but decide not to finally have it in the list of the saved bootnodes. Localstorage is updated only when `Save` is pressed, no matter of the action that the user is doing since last save.

Reply:
> Localstorage is updated only when Save is pressed, no matter of the action that the user is doing since last save.

But why? Can't it just save whenever you add or remove a bootnode?

Reply:
I mean it can - but that makes `Save` obsolete.. 
I can make that change, but that means I won't be following designs and flow logic given on figma.